<ui version="4.0" >
 <class>CLlconServerDlgBase</class>
 <widget class="QDialog" name="CLlconServerDlgBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>588</width>
    <height>438</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>llcon</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="resources.qrc" >:/png/main/res/mainicon.png</iconset>
  </property>
  <property name="sizeGripEnabled" >
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <widget class="QTreeWidget" name="ListViewClients" >
     <property name="rootIsDecorated" >
      <bool>false</bool>
     </property>
     <column>
      <property name="text" >
       <string>Client IP:Port</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>Name</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>Buffers</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>Jitter Buffer Size</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>Block Size Out/ms</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="CheckBoxRegisterServer" >
     <property name="text" >
      <string>Register Server</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLabel" name="LabelCentralServerAddress" >
       <property name="text" >
        <string>Central Server Address:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="CheckBoxDefaultCentralServer" >
       <property name="text" >
        <string>Default</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="LineEditCentralServerAddress" />
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>Server Info</string>
     </property>
     <layout class="QHBoxLayout" >
      <item>
       <layout class="QVBoxLayout" >
        <item>
         <widget class="QLabel" name="LabelServerName" >
          <property name="text" >
           <string>Name</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="LabelLocationCity" >
          <property name="text" >
           <string>Location: City</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="LabelLocationCountry" >
          <property name="text" >
           <string>Location: Country</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" >
        <item>
         <widget class="QLineEdit" name="LineEditServerName" />
        </item>
        <item>
         <widget class="QLineEdit" name="LineEditLocationCity" />
        </item>
        <item>
         <widget class="QComboBox" name="ComboBoxLocationCountry" />
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="spacing" >
      <number>6</number>
     </property>
     <property name="leftMargin" >
      <number>0</number>
     </property>
     <property name="topMargin" >
      <number>0</number>
     </property>
     <property name="rightMargin" >
      <number>0</number>
     </property>
     <property name="bottomMargin" >
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="TextLabelNameVersion" >
       <property name="text" >
        <string>TextLabelNameVersion</string>
       </property>
       <property name="wordWrap" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="buttonOk" >
       <property name="text" >
        <string>&amp;Close</string>
       </property>
       <property name="autoDefault" >
        <bool>true</bool>
       </property>
       <property name="default" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>buttonOk</sender>
   <signal>clicked()</signal>
   <receiver>CLlconServerDlgBase</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
